src/tests/unit/recipeService/getRecipe.test.ts

```diff
@@ -0,0 +1,43 @@
+import { IRecipe } from '../../../interfaces/recipe';
+import { Recipe } from '../../../models/recipeModel';
+import RecipeService from '../../../services/recipeService';
+
+import {
+  mockBody,
+  mockDate,
+  mockId,
+  mockImage,
+  mockTitle,
+} from '../../mocks/mockRecipe';
+
+import { setupTests } from '../../setupTests';
+
+const recipeService = new RecipeService();
+
+setupTests('getRecipe', () => {
+  test('should return a recipe with specified Id', async () => {
+    const newRecipe = <IRecipe>await new Recipe({
+      ownerId: mockId,
+      title: mockTitle,
+      imageUrl: mockImage,
+      body: mockBody,
+      createdAt: mockDate,
+    }).save();
+    const recipe = await recipeService.getRecipe(newRecipe._id);
+
+    expect(recipe?.ownerId).toBe(mockId);
+    expect(recipe?.title).toBe(mockTitle);
+    expect(recipe?.imageUrl).toBe(mockImage);
+    expect(recipe?.body).toBe(mockBody);
+    expect(recipe?.createdAt).toStrictEqual(mockDate);
+  });
+  describe('should not return recipe', () => {
+    test('when id does not exists', async () => {
+      expect(await recipeService.getRecipe(mockId)).toBeNull();
```

Comment:
another syntax for this is
```ts
await expect(recipeService.getRecipe(mockId)).resolves.toBeNull();
```

Still, I think it's good to separate assertions from actions in tests like so:
```ts
const result = await recipeService.getRecipe(mockId);
expect(result).toBeNull();
```